src/components/icon/icon.md

```diff
@@ -0,0 +1,14 @@
+```jsx
+const icon = require('../../svgs/icon-caution-fill.svg').default;
+<Icon name={icon.id} size="large" currentColor="caution" />
+```
+Sizes
+
+```jsx
+const icon = require('../../svgs/icon-caution-fill.svg').default;
+<div>
```

Comment:
TBH, the icons next to each other do not look great / appealing.

Can we provide some margin within this example? Maybe we can structure the documentation a little bit different to make it more pleasant on the eyes?

Reply:
Yeah, I can just one-off the layout from the component's specific CSS

Reply:
![image](https://user-images.githubusercontent.com/142403/61835249-c8f97b80-ae2f-11e9-9c43-039af5d9c1f3.png)

Reply:
Thanks!